<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1170</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="minimumSize">
           <size>
            <width>681</width>
            <height>351</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_2">
          <property name="minimumSize">
           <size>
            <width>415</width>
            <height>321</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_1">
        <property name="minimumSize">
         <size>
          <width>1131</width>
          <height>161</height>
         </size>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>1000</number>
        </attribute>
        <column>
         <property name="text">
          <string>compiler messages</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>1063</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../build-compiler-Desktop_Qt_5_11_3_GCC_64bit-Debug/run.png</normaloff>../build-compiler-Desktop_Qt_5_11_3_GCC_64bit-Debug/run.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1170</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_project"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="text">
    <string>&amp;open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout_project">
   <property name="text">
    <string>About project</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
